pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServer/RebalanceServerConfigurationOptions/RebalanceServerConfigurationOptionToggleSwitch.tsx

```diff
@@ -26,10 +26,13 @@ import {
 type RebalanceServerConfigurationOptionSwitchProps = {
     option: RebalanceServerOption;
     handleConfigChange: (config: { [key: string]: string | number | boolean }) => void;
+    rebalanceConfig: { [optionName: string]: string | boolean | number };
 }
 export const RebalanceServerConfigurationOptionSwitch = (
-    { option, handleConfigChange }: RebalanceServerConfigurationOptionSwitchProps) => {
-    const [isChecked, setIsChecked] = useState<boolean>(option.defaultValue as boolean);
+    { option, handleConfigChange, rebalanceConfig }: RebalanceServerConfigurationOptionSwitchProps) => {
+    const [isChecked, setIsChecked] = useState<boolean>(
+        (Object.keys(rebalanceConfig).includes(option.name) ? rebalanceConfig[option.name] : option.defaultValue) as boolean
+    );
```

Comment:
what are these changes for?

Reply:
This is for the back button. Once back button is hit the configuration page loads and the config values go back to default and don't use the values modified inside `rebalanceConfig`